xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -9896,7 +9896,7 @@ There is a difference in the behavior of the `GetLastWin32Error` method on .NET
 ## Remarks  
  <xref:System.Runtime.InteropServices.Marshal.StringToCoTaskMemAuto%2A> is useful for custom marshaling or for use when mixing managed and unmanaged code. Because this method allocates the unmanaged memory required for a string, always free the memory by calling <xref:System.Runtime.InteropServices.Marshal.FreeCoTaskMem%2A>. This method provides the opposite functionality of <xref:System.Runtime.InteropServices.Marshal.PtrToStringAuto%2A?displayProperty=nameWithType>.  
   
- The characters of the string are copied as either ANSI or Unicode characters, depending on the operating system where the code is executing. On Windows 98, the characters are copied as ANSI characters. On Windows NT 4.0, Windows 2000, Windows XP, and the Windows Server 2003 family, the characters are copied as Unicode characters.  
+ The characters of the string are copied as either ANSI or Unicode characters, depending on the operating system where the code is executing. On Windows NT 4.0, Windows 2000, Windows XP, the Windows Server 2003 family, and later operating systems, the characters are copied as Unicode characters.  
```

Comment:
"On Windows NT 4.0, Windows 2000, Windows XP, the Windows Server 2003 family, and later operating systems" covers all versions of Windows in market today. Remove this wording entirely?